### DISCUSSION TITLE
blog/apakah-neovim-benar-benar-lebih-baik-daripada-vscode

### DISCUSSION THREAD

**Discussion by giscus** [alfariiizi/alfarizi#1] [Q&A]
# blog/apakah-neovim-benar-benar-lebih-baik-daripada-vscode

Opini pribadi mengenai neovim dan bagaimana perbandingannya dengan modern code editor seperti vscode

http://localhost:3000/blog/apakah-neovim-benar-benar-lebih-baik-daripada-vscode

**Comment by alfariiizi** [comment]
Test comments
